cli/remotebazel/remotebazel.go

```diff
@@ -60,6 +60,7 @@ var (
 	execOs         = remoteFlagset.String("os", "linux", "If set, requests execution on a specific OS.")
 	execArch       = remoteFlagset.String("arch", "amd64", "If set, requests execution on a specific CPU architecture.")
 	containerImage = remoteFlagset.String("container_image", "", "If set, requests execution on a specific runner image. Otherwise uses the default hosted runner version. A `docker://` prefix is required.")
+	envStr         = remoteFlagset.String("env", "", "Environment variables to set in the runner environment. Key-value pairs should be separated by '=', and multiple pairs should be separated by ','. Ex. --env=k1=val1,k2=val2")
```

Comment:
- Might be convenient to support omitting `=val`, and in that case get the value using `os.Getenv`. (I think this is the syntax bazel uses for `--action_env`)
- Instead of remoteFlagSet.String, we can use a slice: `flag.New(remoteFlagSet, "runner_env", []string{}, "<usage>")` - that way that the value can be passed more than once, similar to how Bazel's `--action_env` works. This also allows env vals to contain commas (people try to pass JSON objects as auth tokens)

Reply:
Good ideas - done